cpu/native/include/netdev2_tap.h

```diff
@@ -7,66 +7,67 @@
  */
 
 /**
- * @defgroup    dev_eth_tap     Ethernet driver for TAP interfaces
- * @ingroup     native_cpu
+ * @ingroup     netdev2
  * @brief       Low-level ethernet driver for native tap interfaces
  * @{
  *
  * @file
- * @brief       Definitions for @ref net_dev_eth driver for host system's
+ * @brief       Definitions for @ref netdev2 ethernet driver for host system's
  *              TAP interfaces
  *
  * @author      Kaspar Schleiser <kaspar@schleiser.de>
  */
-#ifndef DEV_ETH_TAP_H
-#define DEV_ETH_TAP_H
+#ifndef NETDEV2_TAP_H
+#define NETDEV2_TAP_H
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 #include <stdint.h>
-#include "net/dev_eth.h"
+#include "net/netdev2.h"
 
-#include "net/if.h"
+#include "net/ethernet/hdr.h"
 
 #ifdef __MACH__
 #include "net/if_var.h"
+#else
+#include "net/if.h"
```

Comment:
Shouldn't these includes be in `<...>` instead of `"..."` because they are system headers?